rustual-boy-cli/src/argparse.rs

```diff
@@ -0,0 +1,32 @@
+use clap::{App, Arg};
+
+pub struct CommandLineConfig {
+    pub rom_path: String,
+    pub sram_path: String,
+}
+
+pub fn parse_args() -> CommandLineConfig {
+    let app = App::new("Rustual-boy")
+        .version("0.1.0")
+        .author(crate_authors!(", "))
+        .about("A CLI frontend to the rustual-boy emulator")
+        .arg(Arg::with_name("ROM")
+             .help("The name of the ROM to load")
+             .required(true)
+             .index(1)
+        ).arg(Arg::with_name("SRAM")
+              .help("Path to an SRAM")
+              .short("s")
+              .long("sram")
+        );
+
+    let matches = app.get_matches();
+
+    CommandLineConfig {
+        rom_path: matches.value_of("ROM").unwrap().into(),
```

Comment:
nit: While I don't think it's super important now, I'd like to try and remove all `unwrap` and other improper error handling when we have the chance (esp. with new code). Ideally, this would mean that this fn would end up returning a `Result<CommandLineConfig, some error type>`, and even if we `unwrap` at the top-level in `main`, at least we wouldn't crash in some undescript handler without any additional info or hints to where it is in the repo. Adding as "nit" since I don't think that's a dealbreaker here, but I think it'd be nice (esp. now that we have some nicer syntax with `?`) :)